src/test/scala/circtTests/stage/ChiselStageSpec.scala

```diff
@@ -1095,6 +1095,22 @@ class ChiselStageSpec extends AnyFunSpec with Matchers with chiselTests.Utils {
       info(s"'$expectedOutput' exists")
     }
 
+    it("should emit btor2 to string") {
+      import chisel3._
+      import chisel3.ltl._
+
+      class Counter extends Module {
+        val count = RegInit(0.U(32.W))
+        when(count === 42.U) { count := 0.U }
+        .otherwise { count := count + 1.U }
+        
+        AssertProperty((count <= 42.U))
+      }
+
+      val btor2 = ChiselStage.emitBtor2(new ChiselStageSpec.Foo)
+      println(btor2)
```

Comment:
you can check the contents from btor2, but please don't print it out

Reply:
Yes the goal is to check the contents, I'm just printing it for now for debugging purposes (which is why the PR is still a draft).